Task2/src/interfaces.ts

```diff
@@ -0,0 +1,54 @@
+export interface IWindow extends Window {
+    initMap: () => void;
+}
+
+export interface IDot {
+    lat: number,
+    lng: number
+}
+
+export interface IWeatherResponse {
+    cod: string,
+    count: number,
+    list: [IWeatherItem],
+    message: string
+}
+
+export interface IWeatherItem {
```

Comment:
Please keep interfaces definition shorter. 
You could refactor it by creating separate interfaces for all complecated properties. 

For example, 
create 
`
 IMainWeatherInfo {
        humidity: number,
        pressure: number,
        temp: number,
        temp_max: number,
        temp_min: number
}
`
and then use it in IWeatherItem interface like 

`
interface IWeatherItem { 
...
main:  IMainWeatherInfo 
...
}
`